frontend/lib/norent/letter-builder/steps.tsx

```diff
@@ -74,6 +87,12 @@ export const getNoRentLetterBuilderProgressRoutesProps = (): ProgressRoutesProps
           shouldBeSkipped: isUserInNYC,
           component: NorentLbAskNationalAddress,
         },
+        {
+          path: routes.laAddress,
+          exact: true,
+          shouldBeSkipped: isUserOutsideLA,
+          component: NorentLbLaRedirect,
```

Comment:
I wonder if it makes sense to call this `NorentLbLosAngelesRedirect` just to make it easier for folk who might be unfamiliar w/ the context to understand?  Also I like the idea of being able to do a code search for "losangeles" and find code related to LA, since obviously searching the code for "la" would yield way too many false positives.